### ISSUE TITLE
Cypress 6 support? 

### ISSUE THREAD

**Issue by PascalvdHoek** [dan-cooke/cypress-autostub#6]
Are there any plans soon to support Cypress 6.*?

**Comment by dan-cooke** [comment]
@PascalvdHoek Currently no - but if anyone wants to open a PR it would be a simple enough change! :) 

I may get round to it when im free!

**Comment by dan-cooke** [comment]
It has been a while since I have used cypress, 

In my next project I will be using it more so I will reevaluate if this library should be deprecated in the cypress > 6 world.

especially in later versions of cypress, I am aware they have completely rewritten their mocking code

**Comment by MikeMcC399** [comment]
@dan-cooke 

Should this still be listed on https://docs.cypress.io/plugins ? If it is only relevant for Cypress 5 and earlier, then it will probably not have any new users, especially considering the jump from legacy Cypress 9 to current configuration versions 10, 11 and 12 which are already on the streets.

Maybe it is time to archive it and remove it from the plugins list?

**Comment by dan-cooke** [comment]
@MikeMcC399 yeah absolutely PRs welcome - this library is deprecated and most likely there are much better solutions in modern cypress anyway, last commit is over 3 years ago
